packages/back-end/src/models/FeatureRevisionModel.ts

```diff
@@ -210,18 +231,23 @@ export async function createRevision({
       rules,
     }),
   };
-
+  let status = "draft";
+  if (publish && !requiresReview) {
+    status = "published";
+  } else if (publish && requiresReview) {
+    status = "pending-review";
+  }
   const doc = await FeatureRevisionModel.create({
     organization: feature.organization,
     featureId: feature.id,
     version: newVersion,
     dateCreated: new Date(),
     dateUpdated: new Date(),
-    datePublished: publish ? new Date() : null,
+    datePublished: publish && requiresReview ? new Date() : null,
     createdBy: user,
     baseVersion: baseVersion || feature.version,
-    status: publish ? "published" : "draft",
-    publishedBy: publish ? user : null,
+    status,
+    publishedBy: publish && requiresReview ? user : null,
```

Comment:
Same here, switch to checking if the status is `published` so we can this "requiresReview" logic in one place.